app/controllers/code_schools_controller.rb

```diff
@@ -7,4 +7,8 @@ def index
     @code_schools_by_state = code_schools.by_state
     @country = Country.named('United States')
   end
-end
+
+  def map
+    @code_schools = CodeSchools.new.lat_longs
```

Comment:
This name is kind of confusing when looking at it in the view. Can you rename this to either @code_schools_lat_longs, or just @lat_longs?